third_party/terraform/tests/resource_pubsub_topic_test.go

```diff
@@ -50,3 +73,23 @@ resource "google_pubsub_topic" "foo" {
 }
 `, topic, key, value)
 }
+
+func testAccPubsubTopic_cmek(pid, topicName, kmsKey string) string {
+	return fmt.Sprintf(`
+data "google_project" "project" {
+  project_id = "%s"
+}
+
+resource "google_project_iam_member" "kms-project-binding" {
```

Comment:
As-is, there's a race condition between the topic and this permission. While we could add this dependency with `depends_on`, I'd rather remove it altogether and just rely on the user having configured it in their project. Alternatively, we could add the permission during the `kmsKeyBootstrap` function but that isn't necessary.

Reply:
Since I create projects specifically for the acceptance test to not leave the KMS keys behind, I would prefer to leave the dependency in. I just pushed a change that adds an implicit dependency making the topic project be the IAM member binding, which seems to address the race condition for me. Can we keep this, or does it still trigger the race condition for you (due to the IAM change propagation)?

Reply:
Our CI system is a single project so there's a risk of a race condition here, but I don't think the risk is enough to block on. It'll be resolved on next run anyways, so if we actually encounter this flake we can resolve the issue.